tests/widgets/spinner.spec.coffee

```diff
@@ -36,7 +36,7 @@ describe 'Spinner:', ->
 					m.chai.expect(@stdout.data).to.equal('')
 
 					@spinner.start()
-					clock.tick(60 * 4)
+					clock.tick(60 * 3)
```

Comment:
This is due to https://github.com/helloIAmPau/node-spinner/commit/e6ce44263646c1664dc209b6610f8403d07c5319 where the first iteration is written immediately (so there's immediate spinner output rather than waiting for the first delay to pass)